content/en/blog/_posts/2022-11-28-kubelet-credential-providers/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: 'Kubernetes v1.26: GA Support for Kubelet Credential Providers'
+date: 2022-11-28
```

Comment:
```sugestion
date: 2022-12-22
```
Please update the path `content/en/blog/_posts/2022-11-28-kubelet-credential-providers/index.md` to match.